### DISCUSSION TITLE
Set Telegram user and password

### DISCUSSION THREAD

**Discussion by jackyteam2302** [auanasgheps/snapraid-aio-script#34] [Q&A] [answered]
Hello! 
I have a question about the telegram integration.  i see no password field for the config.

Is it possible to set a password for the telegram bot?  Without it everyone in the world can access the bot and read the information.
Or is there another way to secure the chat that i don't know.

Kind regards

**Comment by auanasgheps** [comment, marked as answer]
Hi,
The telegram integration uses the bot API link, which is private and you receive when you create a bot. This is used to post messages.
Messages are posted to the chat you want (your chat for example), so your information is kept private.

**Reply by jackyteam2302** [reply]
look here, only the API - TOKEN makes the bot not private.  

https://sarafian.github.io/low-code/2020/03/24/create-private-telegram-chatbot.html

**Reply by jackyteam2302** [reply]
If i know the name of your bot, i can find it in telegram and get the same information as you get.

**Reply by jackyteam2302** [reply]
aahh okay i saw it. with the chat id it's save.

**Reply by auanasgheps** [reply]
Correct, if you know the name of the bot you can reach it, but will do nothing. It only sends messages to the chatID you configure in the script.
